app/controllers/api/account.php

```diff
@@ -2353,13 +2353,22 @@
                 // Use current session for events.
                 $queueForEvents->setPayload($response->output($session, Response::MODEL_SESSION));
             }
+
+            $queueForEvents
+                ->setEvent('users.[userId].sessions.[sessionId].delete')
+                ->setParam('userId', $user->getId())
+                ->setParam('sessionId', $session->getId())
+                ->setPayload($response->output($session, Response::MODEL_SESSION))
+                ->trigger();
         }
 
         $dbForProject->deleteCachedDocument('users', $user->getId());
 
         $queueForEvents
+            ->reset()
+            ->setEvent('users.[userId].sessions.delete')
```

Comment:
Dow e need this event?

Reply:
@loks0n Maybe we can skip it for now, as we don't allow bulk deleting of other individual resources, but just "containers" like buckets or collections.